client/src/components/YourOutfit/OutfitCard.jsx

```diff
@@ -67,13 +58,13 @@ const OutfitCard = (props) => {
       <Card className={classes.root} variant="outlined">
         <Grid container>
           <Grid item align="right" xs={12}>
-            <Icon onClick={() => props.handleDeleteOutfitClick(props.product.id)} sx={{ fontSize: 10 }} className={classes.icon}>highlight_off</Icon>
+            <Icon onClick={() => handleDeleteOutfitClick(product.id)} sx={{ fontSize: 10 }} className={classes.icon}>highlight_off</Icon>
           </Grid>
         </Grid>
         <CardMedia
           component='div'
           className={classes.media}
-          image={outfitCardPhoto.photos[0].thumbnail_url || ''}
+          image={outfitCardPhoto.photos[0].thumbnail_url || 'https://media.istockphoto.com/photos/hangers-on-pole-picture-id91001400?b=1&k=20&m=91001400&s=170667a&w=0&h=Ni2lbk6vK2n9PuYSjZ0oR-Vsqfosr1UYuDuHhYmSboE='}
```

Comment:
If this is going to be used a 'default' image, I'm wondering if we can download this image as a local file, to help minimize load time... It's not a huge photo, so It might not matter.... really just an idea =)

Reply:
Would you be using it, too? Maybe it would be worth it!